### PR TITLE
Make Prow components use ghproxy.

### PR DESCRIPTION
**Pull request by cjwagner**
This depends on #1426 (first commit is from that PR).
/hold

/assign @fejta @utka @geeknoid

### CODE REVIEW

**Comment by fejta** [comment]
ref https://github.com/istio/test-infra/issues/1396

**Review by fejta — APPROVED** [review]
/lgtm

File: prow/Makefile [hunk 1]
```diff
@@ -36,6 +36,10 @@ branch-protector-cronjob:
 deck-deployment: get-cluster-credentials
```
Comment (fejta) [inline prow/Makefile:36]: LOL oh there's a makefile! I've just been running kubectl manually

**Review by fejta — APPROVED** [review]
/lgtm

**Comment by cjwagner** [comment]
Updated the Makefile as well.

This is now deployed and seems to be working fine.

**Pull request merged**